src/components/TransactionDetails/index.js

```diff
@@ -535,7 +549,7 @@ export default function TransactionDetails(props) {
     }
     //update the actual slippage value in parent
     updateSlippage(Number(slippageValue))
-  }
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
again i don't think we need this?

Reply:
Yep removed this by adding correct dependencies